tests/tools.vitruv.applications.pcmumlclass.tests/META-INF/MANIFEST.MF

```diff
@@ -4,11 +4,15 @@ Bundle-Name: Vitruv PCM-UML Class Application Tests
 Bundle-SymbolicName: tools.vitruv.applications.pcmumlclass.tests
 Bundle-Version: 0.2.0.qualifier
 Automatic-Module-Name: tools.vitruv.applications.pcmumlclass.tests
-Fragment-Host: tools.vitruv.applications.pcmumlclass
```

Comment:
`pcmumlclass.tests` is no longer a fragment to allow proper dependencies on it.

Reply:
This is fine for now and should be reverted as soon as we finally cleanup all the uilities (see #3).